usefulUtilities/sitV2/appResources/appData/sitClientV2.js

```diff
@@ -370,16 +332,17 @@
         }
 
         // RESET OVERLAYS FOR ALL AVATARS IN RANGE OF THE CHAIR
-        Script.setTimeout(function () { // wait till avatar is out of range of the chair
+        Script.setTimeout(function () {
             Entities.callEntityServerMethod(
                 _this.entityID,
                 "addAllOtherSittableOverlays",
                 AvatarList.getAvatarsInRange(_this.seatCenterPosition, CAN_SIT_M)
             );
-            // ENABLE JUMP DRIVE KEYS
-            for (var i in REMAPPED_DRIVE_KEYS_DURING_SIT) {
-                MyAvatar.enableDriveKey(REMAPPED_DRIVE_KEYS_DURING_SIT[i]);
+            // Enable movement again
+            for (var i in DISABLED_DRIVE_KEYS_DURING_SIT) {
+                MyAvatar.enableDriveKey(DISABLED_DRIVE_KEYS_DURING_SIT[i]);
             }
+            Controller.actionEvent.disconnect(onActionEvent);
```

Comment:
Don't you want to do this _before_ re-enabling the drive keys?

Reply:
I guess there is an extremely slim chance that the space bar could be hit again. I will move it.